Airports/EGLL/SMR/Geo.txt

```diff
@@ -1423,9 +1384,6 @@ N051.28.00.352 W000.26.43.257 N051.28.00.205 W000.26.42.989 smrLLTaxiway
 N051.28.00.205 W000.26.42.989 N051.28.00.106 W000.26.42.721 smrLLTaxiway
 N051.28.00.106 W000.26.42.721 N051.28.00.017 W000.26.42.386 smrLLTaxiway
 N051.28.00.017 W000.26.42.386 N051.27.59.949 W000.26.41.925 smrLLTaxiway
-
-;Taxiway L
```

Comment:
No need to remove these somments as they are stripped by the new complie script

Reply:
These have been added back in as before